### DISCUSSION TITLE
I accidentally deleted a team with a domain in it :(

### DISCUSSION THREAD

**Discussion by norbusonam** [vercel/community#6791] [Account Issue] [answered]
### Question

Like an idiot, I accidentally deleted a team with a domain I was paying for in it. It's now nowhere on my remaining hobby team. Can someone help me get it back?

Thanks!

**Comment by swarnava** [comment]
Have you tried adding the domain in your hobby account manually?

**Reply by norbusonam** [reply]
Yes, it says "The entered domain is not transferable (reason: "Domain already exists in Vercel Inc's account")."

**Reply by amyegan** [reply]
That error make it seem like you're trying to transfer in the domain. If you go to [vercel.com/account/domains](https://vercel.com/account/domains) and just add the domain there, do you see the same error? 

https://vercel.com/docs/projects/domains/add-a-domain#add-and-configure-domain

**Reply by norbusonam** [reply]
When I click add domain I see this:

![image](https://github.com/vercel/community/assets/43245807/7909cf45-b2c6-4ccd-828c-7518e58993d5)

**Reply by amyegan** [reply, marked as answer]
If you don't have any projects yet, try creating a new so you can add the domain to it. You can use one of the free [templates](https://vercel.com/templates) if you need a placeholder. That will let you add the domain back to your account, and you'll be able to move it to a different project later.

**Reply by norbusonam** [reply]
I have it back after starting a new project. I had assumed it would be in domains whether or not it was used in a project. Everything looks to be working now. Thank you!
